src/controllers/projects.js

```diff
@@ -82,21 +82,21 @@ export const deleteProject = async (req, res) => {
   try {
     const result = await Projects.findByIdAndDelete(req.params.id);
 
-    if (!result) {
+    if (!result || !Projects.length) {
```

Comment:
```suggestion
    if (!result) {
```

We can't use !Projects.length 🤔